src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-components.mdx

```diff
@@ -0,0 +1,129 @@
+* `nrk8s-ksm`
+* `nrk8s-kubelet`
+* `nrk8s-controlplane`
+
+The first one is a deployment and the others two are DaemonSets.Each of the components has these containers:
+
+1. A container for the integration, responsible for collecting metrics.
+2. A container with the New Relic infrastructure agent, which is used to send the metrics to New Relic.
+
+<img
+  title="Diagram showing the integration installed in a 3 nodes cluster"
+  alt="Diagram showing the integration installed in a 3 nodes cluster"
+  src={kubernetesv3Integration}
+/>
+
+## Kube-state-metrics component [#nrk8s-ksm]
+
+The cluster state metrics is built on top of the OSS project [`kube-state-metrics`](https://github.com/kubernetes/kube-state-metrics), which is housed under the Kubernetes organization itself. The pod in charge is the one that shares a node with the KSM deployment.
+
+This is how the KSM deployment is scraped:
+
+1. Scraping KSM to a single instance Deployment.
+2. Refactor the code and make it long-running.
+
+A specific `nrk8s-ksm` deployment uses an endpoint informer to locate the IP of the KSM pod and scrape it. The informer caches the list of informers in the cluster locally and watch for new ones.
+
+See [Bring your own KSM](https://github.com/newrelic/helm-charts/blob/master/charts/nri-bundle/README.md#bring-your-own-ksm) for more information about KSM supported versions.
+
+<Callout variant="important">
+  Cloud provider host tags, like `aws.ec2.*`, are no longer included in samples related to entities that are not strictly tied to a particular node: `K8sNamespaceSample`, `K8sDeploymentSample`, `K8sReplicasetSample`, `K8sDaemonsetSample`, `K8sStatefulsetSample`, `K8sServiceSample`, and `K8sHpaSample`.
+  If you need to add an attribute to these samples, use `customAttributes`.
+</Callout>
+
+## Kubelet component [#nrk8s-kubelet]
+
+The Kubelet is the “Kubernetes agent”, a service that runs on every Kubernetes node. It's responsible for creating the containers as instructed by the control plane. Since it's the Kubelet who partners closely with the Container Runtime, it's the main source of infrastructure metrics for our integration, such as use of CPU, memory, disk, network, etc.
+
+`nrk8s-kubelet` runs as a DaemonSet where each instance gathers metric from the Kubelet running in the same node as it is. This component connects to the Kubelet using the Node IP. If this process fails, `nrk8s-kubelet` will fall back to reach the node through the API Server proxy. 
+
+<Callout variant="important">
+  If you have very large clusters, proxying many kubelets might increase the load in the API server.
+</Callout>
+
+To know if the API Server is being used as a proxy, check the logs and look for a similar message to this one:
+
+```
+Trying to connect to kubelet through API proxy
+```
+
+## Control plane component [#nrk8s-controlplane]
```

Comment:
As above

Reply:
I've removed it because @csongnr agrees with your comment

Reply:
repeating the above comment, once again.